boards/v2/qemu/qemu_x86/board.yml

```diff
@@ -0,0 +1,30 @@
+boards:
+
+  - name: qemu_x86
+    socs:
+      - name: ia32
+        variants:
+        - name: 'nokpti'
+        - name: 'nommu'
+        - name: 'nopae'
+        - name: 'virt'
+        - name: 'xip'
+
+  - name: qemu_x86_lakemont
+    socs:
+      - name: lakemont
+
+  - name: qemu_x86_64
+    socs:
+      - name: ia32
+        variants:
+        - name: 'nokpti'
+
+  - name: qemu_x86_tiny
+    socs:
+      - name: ia32
+    revision:
+      format: custom
```

Comment:
with hwmv2 it is now possible to actually implement a board revision check here: https://github.com/zephyrproject-rtos/zephyr/blob/dcd31306f113c1183d0c7d497921d1dd195a9c93/boards/v2/qemu/qemu_x86/revision.cmake#L4-L5 because only boards which specifies `revision: format: custom` will now load this file.

All other boards in this folder / yaml file will ignore the `revision.cmake` file.

Note, this comment is not blocking for this PR, more an extra piece of info.